mne/viz/backends/_pyvista.py

```diff
@@ -159,6 +159,7 @@ def _build(self):
             plotter = self._plotter_class(**self.store)
             plotter.background_color = self.background_color
             self._plotter = plotter
+        # TODO: This breaks trame "client" backend
```

Comment:
unfinished business? seems serious.

Reply:
Only if we try to use the `'client'` backend. I tried it and even removing these lines the rendering was bad for surfaces. I imagine it's worse for volumes. But I do plan on opening an upstream issue